docs/api/websocket.rst

```diff
@@ -35,8 +35,12 @@ middleware objects configured for the app:
 
 .. code:: python
 
+    from typing import Any
+    from falcon.asgi import Request, WebSocket
+
+
     class SomeMiddleware:
-        async def process_request_ws(self, req, ws):
+        async def process_request_ws(self, req: Request, ws: WebSocket) -> None:
```

Comment:
Shouldn't this read `asgi.Request` in line with the above `middleware.rst`?

Reply:
No, I would say I prefer as it is here. But in `middeware.rst` just above we were using this form: `wsgi.Request`, `asgi.Request`, etc.

Reply:
since they should be in different tabs we can probably import the types directly instead of using wsgi. / asgi.

will update

Reply:
This is an Rst file, we can write whatever we want :smiling_imp: